pages/api/hunks.ts

```diff
@@ -0,0 +1,16 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { saveHunk } from '../../utils/db/relevance';
+
+const hunkHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  const hunkJson = req.body;
+  console.info("Saving hunks...");
+  try {
+    await saveHunk(hunkJson);
+  } catch (error) {
+    console.error('Error publishing message:', error);
+    res.status(500).json({ error: 'Failed to publish message. Data must be in the form of a Buffer.' });
```

Comment:
can we have more specific error messages? more specific to the APIs? Since we are using the same error message in the above APIs as well and this will not help in debugging at all I guess. LOW LOGAF though.